### PR TITLE
De-duplication in printer

### PR DESCRIPTION
**Pull request by xilabao**

### CODE REVIEW

**Comment by soltysh** [comment]
/lgtm
@k8s-bot ok to test

File: pkg/printers/humanreadable.go [hunk 1]
```diff
@@ -335,6 +332,19 @@ func appendAllLabels(showLabels bool, itemLabels map[string]string) string {
 	return buffer.String()
 }
 
+// Append a set of tabs for each label column.  We need this in the case where
+// we have extra lines so that the tabwriter will still line things up.
+func AppendLabelTabs(columnLabels []string) string {
```
Comment (janetkuo) [inline pkg/printers/humanreadable.go]: Is anyone calling this function?

Comment (xilabao) [inline pkg/printers/humanreadable.go]: Currently no

**Comment by xilabao** [comment]
@k8s-bot kops aws e2e test this

**Comment by xilabao** [comment]
ping @janetkuo

**Comment by xilabao** [comment]
@k8s-bot non-cri e2e test this
@k8s-bot gce etcd3 e2e test this

**Comment by xilabao** [comment]
ping @soltysh

**Comment by soltysh** [comment]
@xilabao you need to address @janetkuo's comment and wait for her approval.

**Comment by xilabao** [comment]
rebased. @janetkuo @soltysh

**Comment by xilabao** [comment]
@janetkuo kind ping

File: pkg/printers/humanreadable.go [hunk 2]
```diff
@@ -391,6 +388,19 @@ func appendAllLabels(showLabels bool, itemLabels map[string]string) string {
...
```
Comment (janetkuo) [inline pkg/printers/humanreadable.go]: Would you remove this since it's not needed anymore?

**Comment by janetkuo** [comment]
@k8s-bot ok to test 

One comment, lgtm otherwise

File: pkg/printers/humanreadable.go (hunk 2 above)
Comment (xilabao) [inline pkg/printers/humanreadable.go]: done. @janetkuo PTAL

**Review by janetkuo — APPROVED** [review]

**Pull request merged**